jsons/parameters.json

```diff
@@ -15105,6 +15476,40 @@
         "value_translations": {},
         "value_type": "bool"
     },
+    {
+        "default_value": false,
+        "description": "Either to report qvalues in the report. TODO: this parameter has to be revisited, as it is not available in the github documentation!",
+        "key_translations": {
+            "diann_style_1": "report_qvalues_<DROP_KEY>",
+            "ursgal_style_1": "report_qvalues"
+        },
+        "name": "report_qvalues",
+        "tag": [],
+        "triggers_rerun": true,
+        "value_translations": {
+            "diann_style_1": [
+                [
+                    false,
+                    ""
+                ],
+                [
+                    true,
+                    "--qvalue"
+                ]
+            ],
+            "ursgal_style_1": [
+                [
+                    false,
+                    false
+                ],
+                [
+                    true,
+                    true
+                ]
+            ]
+        },
+        "value_type": "bool"
```

Comment:
I know it's not properly documented on the command line reference but this variable is a qvalue filter and takes a float value.

Reply:
so it rather has the shape `--qvalue 0.1`? I just took the flag from the reporter repo, as it did not had any value behind the param ^^